SConstruct

```diff
@@ -937,7 +937,7 @@ def TryScript(config,text,executable):
 
     # Run the given executable with the source file we just built
     output = config.sconf.confdir.File(f + '.out')
-    node = config.env.Command(output, source, executable + " < $SOURCE >& $TARGET")
+    node = config.env.Command(output, source, executable + " $SOURCE > $TARGET")
```

Comment:
This line will be a conflict when you rebase.  The '>&' isn't actually posix compliant, so we fixed that.  However, I'm curious why you removed the '<'.  It's probably fine this way too, but I wouldn't have thought it would be a problem.